test/unit.js

```diff
@@ -66,4 +72,27 @@ describe('Get documents', function() {
       done();
     });
   });
-});
\ No newline at end of file
+});
+
+describe('Register Channel: ', function(){
```

Comment:
I'd say, try mock them out. Then we could run this on travis or something in the future.